src/pretix/plugins/paypal2/templates/pretixplugins/paypal2/checkout_payment_confirm.html

```diff
@@ -2,10 +2,25 @@
 
 <p>
     {% if method == "wallet" %}
-        {% blocktrans trimmed %}
-            The total amount listed above will be withdrawn from your PayPal account after the
-            confirmation of your purchase.
-        {% endblocktrans %}
+        {% if url.url_name == "event.order.pay.confirm" %}
+            <div class="text-center" aria-hidden="true">
+                <span class="fa fa-info-circle fa-4x text-primary"></span>
```

Comment:
I looked at different options for icons, but I guess the info one is the best. Maybe the triangle warning-kind of icon might be an alternative, because it kind of transports a „higher urgency that there is something to do“? So

```suggestion
                <span class="fa fa-exclamation-triangle fa-4x text-primary"></span>
```